cmd/kyma/alpha/sign/sign.go

```diff
@@ -0,0 +1,20 @@
+package sign
+
+import (
+	"github.com/kyma-project/cli/cmd/kyma/alpha/sign/module"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new Kyma CLI command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sign",
+		Short: "Sign all module resources from unsigned component descriptor which hosted in a remote OCI registry",
+		Long:  "Sign all module resources from unsigned component descriptor which hosted in a remote OCI registry",
```

Comment:
why is the long description identical to the short description?

Reply:
```suggestion
		"Use this command to sign all module resources from an unsigned component descriptor that's hosted in a remote OCI registry",
```

Reply:
ok, change to this way, followed the pattern how other command defines, e.g: create command: https://github.com/kyma-project/cli/blob/main/cmd/kyma/alpha/create/create.go#L13-L14